File: EmgenexBusinessPortal/Models/TaskTypeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EmgenexBusinessPortal.Models
{
    public class TaskTypeModel
    {
        public DateTime? CreatedOn = DateTime.UtcNow;
        public DateTime? UpdatedOn = DateTime.UtcNow;
        public int Id { get; set; }
        public string TaskType { get; set; }
        public int BusinessId { get; set; }
        public int CreatedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public string CreatedByName { get; set; }
        public string UpdatedByName { get; set; }
    }
}

File: EmgenexBusinessPortal/BusinessPanel/src/app/models/task-type-model.ts

export class TaskTypeModel {
    constructor(
        public Id: number = 0,
        public TaskType: string = null,
        public CreatedOn: DateTimeFormat = null,
        public UpdatedOn: DateTimeFormat = null,
        public CreatedByName: string = "",
        public UpdatedByName: string = "",        
    ) { }
}

export class TaskTypeFilter {
    constructor(
        public Keywords: string = null,
        public pageSize: number = 25,
        public currentPage: number = 1,
        public SortKey: string = null,
        public SortOrder: string = null,
    ) { }
}
